qiskit/algorithms/gradients/translate_parameterized.py

```diff
@@ -0,0 +1,67 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2022.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Translate parameterized gates only, and leave others as they are."""
+
+from __future__ import annotations
+
+from qiskit.circuit import Instruction, ParameterExpression, QuantumCircuit
+from qiskit.converters import circuit_to_dag
+from qiskit.dagcircuit import DAGCircuit
+from qiskit.transpiler import TransformationPass
+from qiskit.transpiler.preset_passmanagers.common import generate_translation_passmanager
+
+
+class TranslateParameterizedGates(TransformationPass):
```

Comment:
This might be useful to located in `qiskit/transpiler/passes/basis`. Is this specific to gradients?

Reply:
Since it was only used in the gradients I initially put it there, but you're right it might be of more general use! I can move it to the transpiler and add some tests 🙂

Reply:
Done in [f21dfde](https://github.com/Qiskit/qiskit-terra/pull/9067/commits/f21dfdeec3110462324afd94945f724b76814ea4) 🙂